Parsers/ASimDhcpEvent/Parsers/ASimDhcpEventInfobloxBloxOne.yaml

```diff
@@ -0,0 +1,133 @@
+Parser:
+  Title: Dhcp ASIM parser for Infoblox BloxOne
+  Version: '0.1.0'
+  LastUpdated: Jun 21 2024
+Product:
+  Name: Infoblox BloxOne
+Normalization:
+  Schema: Dhcp
+  Version: '0.1'
+References:
+- Title: ASIM Dhcp Schema
+  Link: https://aka.ms/ASimDhcpDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Infoblox BloxOne Documentation
+  Link: https://docs.infoblox.com/space/BloxOneThreatDefense
+Description: |
+  This ASIM parser supports normalizing Dhcp logs from Infoblox BloxOne to the ASIM Dhcp normalized schema. These events are captured through the Azure Monitor Agent (AMA) which are sent by the Data Connector Service of Infoblox BloxOne.
+ParserName: ASimDhcpInfobloxBloxOne
+EquivalentBuiltInParser: _ASim_Dhcp_InfobloxBloxOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: 
+  let EventSeverityLookup = datatable(LogSeverity:string, EventSeverity:string)
+      [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let parser = (disabled:bool=false) {
+      CommonSecurityLog
+      | where not(disabled)
+          and DeviceVendor == "Infoblox"
+          and DeviceEventClassID has "DHCP"
```

Comment:
DeviceEventClassID could be EventOriginalType?

Reply:
Done